showmethekey-gtk/smtk-keys-emitter.c

```diff
@@ -361,7 +361,7 @@ void smtk_keys_emitter_start_async(SmtkKeysEmitter *emitter, GError **error)
 	emitter->cli = g_subprocess_new(
 		G_SUBPROCESS_FLAGS_STDIN_PIPE | G_SUBPROCESS_FLAGS_STDOUT_PIPE |
 			G_SUBPROCESS_FLAGS_STDERR_PIPE,
-		error, "pkexec", INSTALL_PREFIX "/bin/showmethekey-cli", NULL);
+		error, PKEXEC_PATH, INSTALL_PREFIX "/" PACKAGE_BINDIR "/showmethekey-cli", NULL);
```

Comment:
... and here too, make this calculated as an absolute path in meson.

Reply:
I’m not entirely sure I understand you correctly, please look at 1dc36b02b9519a0f48a7415948f551719f29363b.